content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,302 @@
+- [Make Kubernetes aware of the LoadBalancer behaviour](https://github.com/kubernetes/enhancements/issues/1860)
+- [Field `status.hostIPs` added for Pod](https://github.com/kubernetes/enhancements/issues/2681)
+- [Custom profile in kubectl debug](https://github.com/kubernetes/enhancements/issues/4292)
+- [Memory Manager](https://github.com/kubernetes/enhancements/issues/1769)
+- [Support to size memory backed volumes](https://github.com/kubernetes/enhancements/issues/1967)
+- [Improved multi-numa alignment in Topology Manager](https://github.com/kubernetes/enhancements/issues/3545)
+- [Add job creation timestamp to job annotations](https://github.com/kubernetes/enhancements/issues/4026)
+- [Add Pod Index Label for StatefulSets and Indexed Jobs](https://github.com/kubernetes/enhancements/issues/4017)
+- [Auto remove PVCs created by StatefulSet](https://github.com/kubernetes/enhancements/issues/1847)
+    
+    
+### Deprecations and removals
+
+As Kubernetes develops and matures, features may be deprecated, removed, or replaced with better ones for the project's overall health. 
+See the Kubernetes [deprecation and removal policy](/docs/reference/using-api/deprecation-policy/) for more details on this process.
+    
+#### Withdrawal of the old DRA implementation
+
+The enhancement [#3063](https://github.com/kubernetes/enhancements/issues/3063) introduced Dynamic Resource Allocation (DRA) in Kubernetes 1.26.
+
+However, in Kubernetes v1.32, this approach to DRA will be significantly changed. Code related to the original implementation will be removed, leaving KEP [#4381](https://github.com/kubernetes/enhancements/issues/4381) as the "new" base functionality. 
+
+The decision to change the existing approach originated from its incompatibility with cluster autoscaling as resource availability was non-transparent, complicating decision-making for both Cluster Autoscaler and controllers. 
+The newly added Structured Parameter model substitutes the functionality.
+
+This removal will allow Kubernetes to handle new hardware requirements and resource claims more predictably, bypassing the complexities of back and forth API calls to the kube-apiserver.
+
+Please also see the enhancement issue [#3063](https://github.com/kubernetes/enhancements/issues/3063) to find out more.
+
+#### Deprecation of gitRepo volume types
+
+The [gitRepo](https://kubernetes.io/docs/concepts/storage/volumes/#gitrepo) volume type is deprecated and will be removed in a future release, the deprecation has been executed in light of the security advisory encompassing the [CVE-2024-10220](https://nvd.nist.gov/vuln/detail/CVE-2024-10220): Arbitrary command execution through gitRepo volume, which was reported publicly in [this issue](https://github.com/kubernetes/kubernetes/issues/128885).
+    
+#### API removals
+
+There is only a single API removal planned for [Kubernetes v1.32](/docs/reference/using-api/deprecation-guide/#v1-32):
```

Comment:
```suggestion
There is one API removal in [Kubernetes v1.32](/docs/reference/using-api/deprecation-guide/#v1-32):
```

Reply:
Blog is for 1.32, removal will no longer be planned

Reply:
Fixed in the next commit!